src/wp-admin/js/word-count.js

```diff
@@ -66,6 +96,15 @@
 		l10n: window.wordCountL10n || {}
 	};
 
+	/**
+	 * Count words
+	 *
+	 * @memberof wp.utils.wordcounter
+	 *
+	 * @param {string} text - Text to count words in.
```

Comment:
Agreed, see the documented format at https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#functions

(This is a valid JSDoc option but is not used by WP)